pkg/virt-controller/services/virtiofs.go

```diff
@@ -154,22 +154,14 @@ func virtioFSMountPoint(volume *v1.Volume) string {
 	return volumeMountPoint
 }
 
-func generateContainerFromVolume(config *virtconfig.ClusterConfig, volume *v1.Volume, image string, resources k8sv1.ResourceRequirements) k8sv1.Container {
+func generateContainerFromVolume(volume *v1.Volume, image string, resources k8sv1.ResourceRequirements) k8sv1.Container {
 
 	socketPathArg := fmt.Sprintf("--socket-path=%s", virtiofs.VirtioFSSocketPath(volume.Name))
 	sourceArg := fmt.Sprintf("--shared-dir=%s", virtioFSMountPoint(volume))
-	args := []string{socketPathArg, sourceArg, "--cache=auto"}
 
-	securityProfile := restricted
-	sandbox := "none"
-	if virtiofs.CanRunWithPrivileges(config, volume) {
-		securityProfile = privileged
-		sandbox = "chroot"
-		args = append(args, "--xattr")
-	}
+	args := []string{socketPathArg, sourceArg, "--sandbox=none", "--cache=auto"}
 
-	sandboxArg := fmt.Sprintf("--sandbox=%s", sandbox)
-	args = append(args, sandboxArg)
+	securityProfile := restricted
```

Comment:
remove this variable, since we don't change the sec. profile anymore

Reply:
Done.